ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch

```diff
@@ -557,7 +558,7 @@ index 0000000..61bf490
 +          AV_OPT_TYPE_CONST, { .i64 = 1 },  INT_MIN, INT_MAX, VE, "tune" },
 +        { "vmaf", "VMAF optimized mode", 0,
 +          AV_OPT_TYPE_CONST, { .i64 = 2 },  INT_MIN, INT_MAX, VE, "tune" },
-+
++    { "hdr", "High dynamic range input", OFFSET(high_dynamic_range), AV_OPT_TYPE_INT, { .i64 = 0}, 0, 1, VE, "hdr" },
```

Comment:
Could you correct the indent and format(two lines)?

Reply:
`High dynamic range input` is not very accurate, since `-hdr` only signals HDR10 input.
I suggest make it more specific.

Reply:
Jun .. 
Indentation is appropriate when you compare more of the file (lines above the change are for tune/sub-options which are indented relative to tune)

    { "profile", "Profile setting, Main Still Picture Profile not supported", OFFSET(profile),
      AV_OPT_TYPE_INT, { .i64 = FF_PROFILE_HEVC_MAIN }, FF_PROFILE_HEVC_MAIN, FF_PROFILE_HEVC_REXT, VE, "profile"},

    { "qp", "QP value for intra frames", OFFSET(qp),
      AV_OPT_TYPE_INT, { .i64 = 32 }, 0, 51, VE },

    { "rc", "Bit rate control mode", OFFSET(rc_mode),
      AV_OPT_TYPE_INT, { .i64 = 0 }, 0, 1, VE , "rc"},
        { "cqp", NULL, 0, AV_OPT_TYPE_CONST, { .i64 = 0 },  INT_MIN, INT_MAX, VE, "rc" },
        { "vbr", NULL, 0, AV_OPT_TYPE_CONST, { .i64 = 1 },  INT_MIN, INT_MAX, VE, "rc" },

    { "sc_detection", "Scene change detection", OFFSET(scd),
      AV_OPT_TYPE_BOOL, { .i64 = 1 }, 0, 1, VE },

    { "thread_count", "Number of threads [0: Auto, 96: Min]", OFFSET(thread_count),
      AV_OPT_TYPE_INT, {.i64 = 0}, 0, INT_MAX, VE },

    { "tier", "Set tier (general_tier_flag)", OFFSET(tier),
      AV_OPT_TYPE_INT, { .i64 = 0 }, 0, 1, VE, "tier" },
        { "main", NULL, 0, AV_OPT_TYPE_CONST, { .i64 = 0 }, 0, 0, VE, "tier" },
        { "high", NULL, 0, AV_OPT_TYPE_CONST, { .i64 = 1 }, 0, 0, VE, "tier" },

    { "tune", "Quality tuning mode", OFFSET(tune), AV_OPT_TYPE_INT, { .i64 = 1 }, 0, 2, VE, "tune" },
        { "sq", "Visually optimized mode", 0,
          AV_OPT_TYPE_CONST, { .i64 = 0 },  INT_MIN, INT_MAX, VE, "tune" },
        { "oq",  "PSNR / SSIM optimized mode",  0,
          AV_OPT_TYPE_CONST, { .i64 = 1 },  INT_MIN, INT_MAX, VE, "tune" },
        { "vmaf", "VMAF optimized mode", 0,
          AV_OPT_TYPE_CONST, { .i64 = 2 },  INT_MIN, INT_MAX, VE, "tune" },
    { "hdr", "High dynamic range input", OFFSET(high_dynamic_range),AV_OPT_TYPE_INT, { .i64 = 0}, 0, 1, VE, "hdr" },
    {NULL},
};

Reply:
I recommend changing description to "High dynamic range input (HDR10)"
Would this be adequate?

Reply:
Sure. Let's go with it.